src/routes/business/authorizedPerson.ts

```diff
@@ -0,0 +1,79 @@
+import type { Response } from "express";
+
+import generateID from "../../helpers/id";
+import uuid from "node-uuid";
+import { AuthorizedPerson } from "../../helpers/types";
+import { saveBusiness, getBusinessIdByAccount } from "../../db";
+import { RequestWithBusinessAndAccount } from "../../helpers/middlewares";
+
+export const createAuthorizedPerson = async (
+  req: RequestWithBusinessAndAccount,
+  res: Response
+) => {
+  const { business } = req;
+  const { account_id } = req.params;
+
+  const authorizedPersonId = req.body.authorized_person_id;
+  if (!authorizedPersonId) {
+    return res.status(400).send({
+      errors: [
+        {
+          id: uuid.v4(),
+          status: 400,
+          code: "missing_field",
+          title: "Missing Field",
+          detail: "Invalid request.",
+        },
+      ],
+    });
+  }
+
+  const businessId = await getBusinessIdByAccount({ id: account_id });
+  if (businessId !== business.id) {
+    return res.status(403).send({
+      errors: [
+        {
+          id: uuid.v4(),
+          status: 403,
+          code: "forbidden",
+          title: "Forbidden",
+          detail: "You are not authorized to perform this action.",
+        },
+      ],
+    });
+  }
```

Comment:
403 is thrown only when the endpoint is not enabled for us. But on Mocksolaris everything is enabled
you do not need to fetch business by account here. The account entity is already stored on business and you can just compare ids of existing account and the one from params. If they do not match 404 should be thrown

Reply:
where does it return null? in the test?